src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,18 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
+    @Size(min = 8)
     private String email;
+    @Size(min = 4)
     private String password;
+    @Size(min = 4)
+    private String repeatPassword;
```

Comment:
@Email and @Password should cover all points including size
```suggestion
    private String email;
    private String password;
    private String repeatPassword;
```